### PR TITLE
[systeminfo] Added validation for pid on channel group process to avoid…

### PR DESCRIPTION
**Pull request by cweitkamp**
…DeviceNotFoundExceptions

Solves issue #2222.

Signed-off-by: Christoph Weitkamp github@christophweitkamp.de

### CODE REVIEW

File: addons/binding/org.openhab.binding.systeminfo/src/main/java/org/openhab/binding/systeminfo/model/OshiSysteminfo.java [hunk 1]
```diff
@@ -29,8 +29,9 @@
 import oshi.util.EdidUtil;
 
 /**
- * This implementation of {@link SysteminfoInterface} is using the open source library OSHI to provide system
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.systeminfo/src/main/java/org/openhab/binding/systeminfo/model/OshiSysteminfo.java]: Please make sure that you use the same formatting or revert this changes manually

**Comment by martinvw** [comment]
@svilenvul do these changes seem okay to you?

**Comment by svilenvul** [comment]
Yes.
